src/metabase/driver/sql_jdbc/sync.clj

```diff
@@ -24,6 +24,7 @@
   excluded-schemas
   fallback-metadata-query
   filtered-syncable-schemas
+  get-tables
```

Comment:
haha yea I did find this and the interface confusing, I wanted to do a bit of refactoring to remove one of them since it makes grepping code hard for these methods.